tests/test_browser.py

```diff
@@ -208,43 +205,30 @@ def test_save_screenshot(self):
         # Check that the file is not empty
         assert os.stat(expected_file).st_size > 100
 
-    def test_save_screenshot_unsupported(self, caplog):
+    def test_save_screenshot_unsupported(self):
```

Comment:
Argh, GitHub lost my edit on this one.  I ran `pytest tests/test_browser.py::TestSaveFiles` and got "cannot collect test class 'TestSaveFiles' because it has a __init__ constructor".  Replacing the constructor with class attributes initialized to `None` got the tests to run, although one of them failed.